Source/SourceKittenFramework/DeclarationType.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  DeclarationType.swift
+//  SourceKitten
+//
+//  Created by Paul Young on 12/4/15.
+//  Copyright © 2015 SourceKitten. All rights reserved.
+//
+
+public protocol DeclarationType {
+    var language: Language { get }
+    var kind: DeclarationKindType? { get }
+    var location: SourceLocation { get }
+    var extent: Range<SourceLocation> { get } // FIXME: Type 'SourceLocation' does not conform to protocol 'ForwardIndexType'
```

Comment:
Cool. I think it's a tuple in the existing `SourceDeclaration` but I was following your lead in https://github.com/jpsim/SourceKitten/issues/87#issuecomment-159002802.